### ISSUE TITLE
Fix regression introduced in #1316: drawer buttons disappear making switching between drawer content impossible

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#1367]
I did some more testing  and I'm not sure we can release this in 3.13. The thing is that **if this option is enabled, there is no way to switch between the different content types in Drawer**. Keep in mind that certain maps (like https://framtidsplan2050.halmstad.se) use Drawer to display contents from both the Tools menu _and_ DocumentHandler. (There are also ideas about displaying legend graphic in a third tab in Drawer.)

So, enabling this option removes also these buttons:
![Skärmavbild 2023-06-29 kl  14 56 13](https://github.com/hajkmap/Hajk/assets/110222/dddd4eb1-a4fc-4c1b-a7d7-661c2d4162e8)

Which unfortunately breaks certain map configurations and user are stuck in this view: 
![Skärmavbild 2023-06-29 kl  14 58 03](https://github.com/hajkmap/Hajk/assets/110222/a6ea4e9c-eb7f-4c39-be1b-2943836d1619)

I can see where it would be desirable to use this option (e.g. don't allow hiding the Drawer) but still be able to switch between Drawer content. 

**As I stated before, this feature is currently half-baked and must be completed before the next release. It is already merged to `develop` so I open a new issue (this) which can be closed upon completion. There's still plenty of time to do this before the next release so I think that we can avoid a revert in this case.**

Ping @OlofSvahnVbg, @Hallbergs, @jesade-vbg.

_Originally posted by @jacobwod in https://github.com/hajkmap/Hajk/issues/1316#issuecomment-1613142389_

**Comment by Hallbergs** [comment]
I agree that the ability to switch drawer content even if the static option is active must be added before releasing it.

**Comment by jesade-vbg** [comment]
I guess this isn't urgent as you need to check a checkbox to activate? In that case it can wait until @OlofSvahnVbg is back. Ok?

**Comment by jacobwod** [comment]
Absolutly, I'm fine with it as long as we get it done before 3.13. 👍
